source/_integrations/roku.markdown

```
@@ -88,27 +121,36 @@ data:
     - select
 ```
 
-## Media player
+#### Select
 
-When the Home Assistant Roku integration is enabled and a Roku device has been configured, in the Home Assistant UI the Roku media player will show a listing of the installed channels, or apps, under “source”. Select one and it will attempt to launch the channel on your Roku device.
+- **Application control**
+  - **Description**: The application select control will allow changing the active application.
+  - **Available for devices**: All 
```

Markdownlint (0.35.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)